code/modules/clothing/outfits/ert.dm

```diff
@@ -379,3 +379,73 @@
 		/obj/item/megaphone/clown = 1,
 		/obj/item/reagent_containers/spray/chemsprayer/janitor/clown = 1,
 		)
+
+/datum/outfit/death_commando
+	name = "Death Commando"
+
+	uniform = /obj/item/clothing/under/syndicate/combat
```

Comment:
Why are they using the syndicate combat suit over the nanotrasen officer suit?

Reply:
The NT Officer suit doesn't have protection and is more like a fancy business suit, and it wouldn't make any difference if I made a new combat jumpsuit with the sole difference being that it doesn't have /syndicate/ in its filepath, something you would never see ingame.

Reply:
powercreep, theres really no point to replacing their jumpsuits

Reply:
> powercreep

This shouldn't apply here since deathsquad is made to be literally undefeatable and broken.

Reply:
They don't have protection since they don't need protection due to their hardsuit providing 80% resistance to all damage types. Also the NT officer suit makes more sense since its NT and the suit worn by ALL nt officials.

Reply:
Also they shouldn't get more armour, since it stacks with the hardsuit

Reply:
Fine.